src/common/modules/UnicodeFontHandler.js

```diff
@@ -0,0 +1,245 @@
+				if (font.length == 26 || font.length == 26 + 10)
+					letter = font[code - 'a'.charCodeAt(0)];
+				else if (font.length == 26 + 26 || font.length == 26 + 26 + 10)
+					letter = font[code - 'a'.charCodeAt(0) + 26];
+			} else if (letter >= '0' && letter <= '9') {
+				if (font.length == 26 + 10)
+					letter = font[code - '0'.charCodeAt(0) + 26];
+				else if (font.length == 26 + 26 + 10)
+					letter = font[code - '0'.charCodeAt(0) + 26 + 26];
+			}
+		}
+		output += letter;
+	}
+
+	return output;
+}
+
+/**
+ * Capitalize Each Word.
+ *
+ * @param {string} atext
+ * @returns {string}
+ */
+function capitalizeEachWord(atext) {
+	// Requires Firefox/Thunderbird 78
+	return atext.replace(/(?<=^|\P{Alpha})\p{Alpha}\S*/gu, ([h, ...t]) => h.toLocaleUpperCase() + t.join(''));
+}
+
+/**
+ * Toggle Case.
+ *
+ * @param {string} atext
+ * @returns {string}
+ */
+function toggleCase(atext) {
+	let output = '';
+
+	for (let letter of atext) {
+		const upper = letter.toLocaleUpperCase();
+		const lower = letter.toLocaleLowerCase();
+		if (letter === lower && letter !== upper)
+			letter = upper;
+		else if (letter === upper && letter !== lower)
+			letter = lower;
+		output += letter;
+	}
+
+	return output;
+}
+
+/**
+ * Change Case
+ *
+ * @public
+ * @const
+ * @type {Object.<string, function>}
+ */
+const changeCase = {
+	"lowercase": (str) => str.toLocaleLowerCase(),
+	"uppercase": (str) => str.toLocaleUpperCase(),
+	"capitalize-each-word": (str) => capitalizeEachWord(str.toLocaleLowerCase()),
+	"toggle-case": (str) => toggleCase(str)
+};
+
+/**
+ * Handle context menu click.
+ *
+ * @param {Object} info
+ * @param {Object} tab
+ * @returns {void}
+ */
+function handle(info, tab) {
+	let text = info.selectionText;
+
+	if (text) {
+		text = text.normalize();
+		let output = '';
+
+		switch (info.menuItemId) {
+			case "lowercase":
+			case "uppercase":
+			case "capitalize-each-word":
+			case "toggle-case":
+				output = changeCase[info.menuItemId](text);
+				break;
+			case "superscript":
+			case "small-caps":
+			case "all-small-caps":
+			case "unicase":
+			case "serif-bold":
+			case "serif-italic":
+			case "serif-bold-italic":
+			case "sans-serif":
+			case "sans-serif-bold":
+			case "sans-serif-italic":
+			case "sans-serif-bold-italic":
+			case "script":
+			case "script-bold":
+			case "fraktur":
+			case "fraktur-bold":
+			case "monospace":
+			case "double-struck":
+			case "circled":
+			case "circled-(black)":
+			case "squared":
+			case "squared-(black)":
+			case "fullwidth": {
+				output = changeFont(text, info.menuItemId);
+				break;
+			}
+		}
+
+		browser.tabs.executeScript(tab.id, {
+			code: `insertIntoPage("${output}");`,
+			frameId: info.frameId
+		});
+	}
+}
+
+/**
+ * Apply new Unicode font settings.
+ *
+ * @param {Object} unicodeFont
+ * @returns {void}
+ */
+function applySettings(unicodeFont) {
+	// Thunderbird
+	const menus = THUNDERBIRD ? browser.menus : browser.contextMenus;
```

Comment:
Actually, if we just use feature-detection, we can simplify that, by specifying it as a fallback:
```suggestion
	const menus = browser.menus || browser.contextMenus; // fallback for Thunderbird
```

Not tested, but this may be a browser-agnostic idea.